### PR TITLE
Create PT phone number Ecto.Type

### PR DESCRIPTION
**Pull request by danielsp45**
Due to the phone numbers having specific formats and rules, there was a need to use [Ecto.Types](https://hexdocs.pm/ecto/Ecto.Type.html) to regulate the PT phone numbers coming into the database.
After some research the [E.164](https://www.twilio.com/docs/glossary/what-e164) format was the best option for this project, which consists of `[+] [country code] [subscriber number including area code]`.

### CODE REVIEW

File: lib/bokken/types/pt_mobile.ex [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    {:error, [message: "número sem indicativo"]}
+  end
+
+
+  # when putting the phone number into the database this function 
+  # transforms the data into a specific format to be stored
+  def dump("+351" <> mobile) do
+    {:ok, "+351" <> mobile}  
+  end
+  def dump(_mobile) do
```
Comment (nelsonmestevao) [inline lib/bokken/types/pt_mobile.ex]:
```suggestion
  def dump(_mobile), do: :error
```

Try this to make it a oneliner 💪

File: lib/bokken/types/pt_mobile.ex [hunk 2]
```diff
@@ -0,0 +1,65 @@
+defmodule Bokken.Types.Ecto.PtMobile do
```
Comment (nelsonmestevao) [inline lib/bokken/types/pt_mobile.ex]: In our application logic we try to follow folder name base rules. For example, each module name has it own folder.

What do you think in changing this to `Bokken.Ecto.PtMobile` and make it in `lib/bokken/ecto/pt_mobile.ex`?

**Review by nelsonmestevao — APPROVED** [review]
Great work! :clap:

File: lib/bokken/types/pt_mobile.ex (hunk 2 above)
Comment (danielsp45) [reply lib/bokken/types/pt_mobile.ex]: Yes I agree. I wasn't sure of where to put this module

**Pull request merged**